components/purchase_modal/process_payment_setup.tsx

```diff
@@ -116,6 +120,10 @@ class ProcessPaymentSetup extends React.PureComponent<Props, State> {
 
             // the action subscribeCloudSubscription returns a true boolean when successful and an error when it fails
             if (typeof productUpdated !== 'boolean') {
+                pageVisited(
+                    TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                    'pageview_payment_failed',
+                );
```

Comment:
What is the difference this event with the one above? Is this some other type of failure ?

Reply:
It’s same since it still goes to the error page. The difference is the it occurs for different requests